docs/Preview to Stable Changelog.md

```diff
@@ -0,0 +1,369 @@
+﻿# PowerShell Preview Extension Release History March 2020

```

Comment:
Maybe worth renaming the file to something like `preview_to_stable_changelog.md`